kubetest/objects/replicaset.py

```diff
@@ -39,8 +39,6 @@ def __init__(self, *args, **kwargs) -> None:
         super(ReplicaSet, self).__init__(*args, **kwargs)
         self._add_kubetest_labels()
 
-        client.AppsV1Api.read_namespaced_replica_set()
```

Comment:
@linkous8 I presume this fails on init?

is there another test that should address this (perhaps it's not even important?)

Reply:
Correct, it raises an error for missing parameters. The updated implementation brings it into line with the other controller models implemented by this library. Eg. :
https://github.com/opsani/kubetest/blob/f77c5c8056d9ad51a699312874b350cff3f77529/kubetest/objects/deployment.py#L39
https://github.com/opsani/kubetest/blob/f77c5c8056d9ad51a699312874b350cff3f77529/kubetest/objects/statefulset.py#L39

As for testing, this library only implements unit testing for model instantiation as well as loading from manifests. I was unable to locate any tests that interact with live objects in a cluster